Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -161,6 +170,10 @@ public final class ProjectPamphletMainCellViewModel: ProjectPamphletMainCellView
         creatorDetails == nil || isLoading
       }
 
+    self.creatorButtonIsHidden = projectAndCreatorDetails.map { _, creatorDetails in
```

Comment:
Could we just make this:

```swift
self.creatorButtonIsHidden = self.creatorStackViewHidden
```
That way we know the exact same behaviour of hiding/showing the stackview is applied to the button.